.github/workflows/others.yml

```diff
@@ -0,0 +1,34 @@
+name: Others CI/CD pipeline
```

Comment:
What is this for?

Reply:
Il we are not changing anything in backend or frontend it will still run naming convention check